sections/findify-filter-group.liquid

```diff
@@ -67,24 +71,29 @@
             , name: name
             , type: type
             , component: component
-            , filters: filters %}
+            , filters: filters
+            , min: min
+            , max: max
+            , sliderFrom: from
+            , sliderTo: to
+          %}
```

Comment:
I suggest to render `findify-filter-range-slider` here, I mean to do smth like this

```liquid
{% when 'range', 'reviews' %}
          {% if name == 'price' %}
            {% render 'findify-filter-range-slider', min: min, max: max, name: name, from: from, to: to %}
          {% else %}
            {% render 'findify-filter-range', name: name, type: type, component: component, filters: filters %}
          {% endif %}
``` 
and initially comment the code out. In the case we dont need to customize `findify-filter-range` and the code is separated

**Implemented** https://yana-test-store.myshopify.com/?_ab=0&_bt=eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaEpJaUo1WVc1aExYUmxjM1F0YzNSdmNtVXViWGx6YUc5d2FXWjVMbU52YlFZNkJrVlUiLCJleHAiOiIyMDI0LTA1LTIzVDEzOjU1OjUyLjg4NVoiLCJwdXIiOiJwZXJtYW5lbnRfcGFzc3dvcmRfYnlwYXNzIn19--d8d8beb04030be86d2d1638ef9f08ae69aa405dc&_fd=0&_sc=1&key=2e7772c8d721f5344e040930235bdbe1cf4e75e32e1e43e5a3b1ad38902a1b53&preview_theme_id=135988019359